notesScript.js

```diff
@@ -16,6 +16,16 @@ function addNote(notes)
         noteDiv.classList.add('note');
```

Comment:
also, can we rename `addNote` to something like `renderNotesList`? `addNote` suggests that it would create a new note or something